### PR TITLE
Fixed sonar pulse revealing shroud

### PR DESCRIPTION
**Pull request by Mailaender**
Partially addresses https://github.com/OpenRA/OpenRA/issues/6145.

### CODE REVIEW

**Comment by pchote** [comment]
I have edited the description, as this does not close the referenced issue.

**Comment by GraionDilach** [comment]
:+1:

File: mods/ra/rules/misc.yaml [hunk 1]
```diff
@@ -352,7 +349,7 @@ powerproxy.sonarpulse:
 		Icon: sonar
 		Description: Sonar Pulse
 		LongDesc: Reveals all submarines in the vicinity for a \nshort time.
-		ChargeTime: 30
+		ChargeTime: 20
```
Comment (pchote) [inline mods/ra/rules/misc.yaml]: This seems excessively fast to me.

Comment (GraionDilach) [inline mods/ra/rules/misc.yaml]: Oh wait, true - this line should be reverted, sonar pulse was good as-is in terms of recharge.

Comment (Mailaender) [inline mods/ra/rules/misc.yaml]: Now that I buffed it to not reveal without other (naval) units I thought that would be okay. Removed the change.

**Comment by abcdefg30** [comment]
:+1: / :white_check_mark:

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/0ef3b42f8eefe00ac8b4d816638a273361db35d7...6edfd2f0f33c9c656b2517eb8db1ad88724b3047)

**Pull request merged**